cfme/tests/infrastructure/test_kubevirt.py

```diff
@@ -3,23 +3,75 @@
 import pytest
 
 from cfme.infrastructure.provider.kubevirt import KubeVirtProvider
+from cfme.provisioning import do_vm_provisioning
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
 from cfme.utils.update import update
 
+
 pytestmark = [
-    pytest.mark.provider([KubeVirtProvider], scope="module")
+    pytest.mark.provider([KubeVirtProvider]),
+    pytest.mark.usefixtures('setup_provider')
 ]
 
 
-def test_k6t_provider_crud(provider):
-    """Tests provisioning via PXE
+@pytest.fixture
+def temp_vm(appliance, provider, provisioning):
+    template_name = provisioning['template']
+    vm_name = random_vm_name('k6tvm')
+    prov_data = {'catalog': {'vm_name': vm_name}}
+    vm = appliance.collections.infra_vms.instantiate(name=vm_name,
+                                                     provider=provider,
+                                                     template_name=template_name)
+    note = ('template {} to vm {} on provider {}'.format(template_name, vm_name, provider.key))
+    prov_data.update({
```

Comment:
Since this fixture is used for a power management test, why not create the VM directly on the provider through its API? This will be faster than provisioning through the UI, and the CRUD test is already covering UI provisioning.

Reply:
Agree, I'll add it with rest option for provider crud in an other PR